Lesson_5/index.html

```diff
@@ -0,0 +1,76 @@
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="style.css" type="text/css">
+    <link href='https://fonts.googleapis.com/css?family=Jaldi' rel='stylesheet'>
+	<title>Mouseover and mouseleave</title>
+</head>
+<body>
+	<div class="buttonBlock">
+	<button id="clickMeButton" class="newclass">My button</button>
+	</div>
+	<script>
+		let clickMeButton = document.getElementById("clickMeButton");
+		const nameOfTheButton = "My button";
```

Comment:
it's not actually the name it's label or text for the button. So the variable name could be miss leading.